cmd/main.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/minio/cli"
 	"github.com/minio/mc/pkg/console"
+	"github.com/minio/minio/pkg/distance"
```

Comment:
I see `words` is good enough the underneath function is `DamerauLevenshteinDistance()`